README.md

```diff
@@ -21,10 +21,131 @@ It implements the following architecture:
 
 
 # :page_facing_up: Dependencies
-* [YARP](http://www.yarp.it/): to handle the comunication with the robot with both ovrheadset and SDLjoypad drivers;
-* [Walking-controllers](https://github.com/robotology/walking-controllers) to handl the hand cartesian task and walking commands
-* [Oculus SDK](https://developer.oculus.com/downloads/package/oculus-sdk-for-windows/): The native Windows SDK.
-* [Cyberith SDK](https://www.cyberith.com/research-development/): To allow the virtualizer module to capture the operator data.
+ 1. **robotology superbuild [optional]:** Install the robotology superbuild on a Windows machine as described here [Windows installation notes](https://github.com/robotology/robotology-superbuild#windows):
+
+    1. opencv: Do not use the opencv package as mentioned there (Dec 2018), instead install it from the [opencv website] (https://opencv.org/releases.html), version 3.4.4;
```

Comment:
Can you explain better ```Do not use the opencv package as mentioned there (Dec 2018)```, You can just add the link at the following issue: https://github.com/robotology/robotology-superbuild/issues/145  
Furthermore there is  there is a space between **]** and **(** ```[opencv website] (https://opencv.org/releases.html)```

Reply:
done.